back/KinoFund/DAL/Migrations/20220129123739_categoryTest.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```

Comment:
remove all init tables migrations (with name test) and create one migration with name InitTables